### DISCUSSION TITLE
Prestashop to Arm M1

### DISCUSSION THREAD

**Discussion by nio74** [ddev/ddev#5917] [Q&A]
Hi, I would like know if anyone has installed with nginx web server, because at me not work it. While If use apache-fpm it work

**Comment by rfay** [comment]
Can you say what problems you have? Many older frameworks are based around Apache and rely on .htaccess to work correctly. I don't know if this might be the case with Prestashop. Do their docs say to use Apache?

**Reply by rfay** [reply]
I see that Prestashop requires Apache, so I guess that is the issue: https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/

**Comment by nio74** [comment]
Whit Apache-fpm it work, but if I wont use how webserver nginx-fpm all links in to backend ponint  to home page in front end.  For example if I want to select menu Design  it show home page.

**Reply by rfay** [reply]
I think you'll have to open an issue with Prestashop asking for nginx support, right? They say you need Apache.
